rubygem-foreman_templates/rubygem-foreman_templates.spec

```diff
@@ -35,6 +35,8 @@ Requires: %{?scl_prefix}rubygem(git)
 BuildRequires: %{?scl_prefix_ruby}ruby(release)
 BuildRequires: %{?scl_prefix_ruby}rubygems-devel
 BuildRequires: %{?scl_prefix_ruby}rubygems
+BuildRequires: foreman-plugin >= 1.15
```

Comment:
oh I though it was assets precompilation, now I see the the same macro is used with precompile flags